toolbox

```diff
@@ -638,6 +744,42 @@ case $op in
         $ls_containers && list_containers
         exit
         ;;
+    rm | rmi )
+        force=false
+        all=false
```

Comment:
Since this is a shell script, and all variables are global, it would be nicer to namespace these variables as `rm_all`, `rm_force`, etc. to prevent them from accidentally leaking into another context and suprising us.

Reply:
Done.